packages/block-library/src/navigation-menu/editor.scss

```diff
@@ -2,6 +2,29 @@
 // @todo: eventually we may add a feature that lets a parent container absorb the block UI of a child block.
 // When that happens, leverage that instead of the following overrides.
 [data-type="core/navigation-menu"] {
+
+	// Allow overflowing items to scroll container
+	.wp-block-navigation-menu > .wp-block-navigation-menu__scroll-container {
+		// if X is "auto" or "scroll" then spec says Y must also assume this value
+		// therefore the Y overflow is effectively "clipped" meaning that the toolbar
+		// of child Blocks cannot display correctly. We "fix" this below.
+		// Note: as Y will be "auto" anyway we need to hide the scroll bars
+		// created by the artificial spacing.
+		overflow-x: auto;
+		overflow-y: hidden; // required for FF
+
+		// Hack: create artificial overflow space within the element which has its Y
+		// value clipped (see above). This value is equal to that of the positioned
+		// toolbar element (usually `top: -51px`).
+		padding-top: 51px;
+		margin-top: -51px;
```

Comment:
Might be nice to add `padding-left: 17px` here to avoid the toolbar on the first nav element being cut off:

<img width="186" alt="Screen Shot 2019-11-13 at 4 41 29 pm" src="https://user-images.githubusercontent.com/8096000/68736610-362a5a00-0635-11ea-82ad-46d3a332526e.png">